source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -12,6 +12,7 @@ struct SpacingDefinition
     unsigned int right = 0;
     unsigned int top = 0;
     unsigned int bottom = 0;
+    SpacingDefinition(int left = 0, int right = 0, int top = 0, int bottom = 0):left(left), right(right), top(top), bottom(bottom){};
```

Comment:
We use the SpacingDefintions and structures defined here as members of other structures, and use initializes in form of sturctname(val, val, ...). 
This causes compilation errors on iOS side. To fix it, I added dedicated constructors and initialize lists with default values.There are other ways to fix it, but it would require editing instead of adding.